src/main/kotlin/racingcar/InputValidatorService.kt

```diff
@@ -0,0 +1,56 @@
+package racingcar
+
+import racingcar.domain.Car
+import racingcar.domain.Messages
+
+class InputValidatorService {
+    fun getValidatedCarNames(input: String): List<Car> {
+        emptyCheck(input)
+        val names = input.split(",").map { it.trim() }
+        duplicateNameCheck(names)
+
+        names.forEach { name ->
+            nameLengthCheck(name)
+            availableNameCheck(name)
+        }
+        return toCars(names)
+    }
+
+    fun getValidatedRaceCount(input: String): Int {
+        emptyCheck(input)
+        val number = positiveInputCheck(input)
+        tryCountLengthCheck(number)
+        return number
+    }
+
+    private fun emptyCheck(input: String) {
+        require(input.isNotBlank()) { Messages.ERROR_EMPTY_INPUT.message }
+    }
+
+    private fun nameLengthCheck(name: String) {
+        require(name.length in 1..5) { Messages.ERROR_NAME_LENGTH.message }
+    }
+
+    private fun duplicateNameCheck(names: List<String>) {
+        require(names.size == names.distinct().size) { Messages.ERROR_DUPLICATE_NAME.message }
+    }
+
+    private fun availableNameCheck(name: String) {
+        val regex = Regex("^[가-힣a-zA-Z0-9]*$")
+        require(regex.matches(name)) { Messages.ERROR_NOT_AVAILABLE_NAME.message }
+    }
+
+    private fun positiveInputCheck(input: String): Int {
+        val number =
+            runCatching { input.toInt() }
+                .getOrElse { throw IllegalArgumentException(Messages.ERROR_NOT_POSITIVE.message) }
```

Comment:
```suggestion
val number = input.toIntOrNull() ?: throw IllegalArgumentException(Messages.ERROR_NOT_POSITIVE.message)
```

이런 형태도 가능해보이는데요! kotlin에는 다양한 확장함수들이 존재하다보니 한 번씩 둘러보고 사용해보시는 것을 추천드립니다 👍